tests/integration/buildcmd/test_build_terraform_applications.py

```diff
@@ -50,20 +60,83 @@ def setUpClass(cls):
             ):
                 LOG.info(log)
 
-    def setUp(self):
-        super().setUp()
-        shutil.rmtree(Path(self.working_dir))
-        shutil.copytree(Path(self.terraform_application_path), Path(self.working_dir))
-
     def run_command(self, command_list, env=None, input=None, timeout=None):
-        process = Popen(command_list, stdout=PIPE, stderr=PIPE, stdin=PIPE, env=env, cwd=self.working_dir)
+        process = Popen(
+            command_list, stdout=PIPE, stderr=PIPE, stdin=PIPE, env=env, cwd=self.terraform_application_execution_path
+        )
         try:
             (stdout, stderr) = process.communicate(input=input, timeout=timeout)
             return stdout, stderr, process.returncode
         except TimeoutExpired:
             process.kill()
             raise
```

Comment:
nit: Why not using the common `runCommand` method that we have? Are there any differences for Terraform tests?

Reply:
I think there was a reason initially but I can't remember what that is. I don't think we need it anymore but we've already coupled many of our test cases to this, so I'm just going to use this method as a wrapper to call our existing `run_command()` (since the cases expect a slightly different response).